Mutagen.Bethesda.Analyzers.Cli/Modules/AnalyzerCommandModule.cs

```diff
@@ -72,5 +72,14 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<NullPluginListingsPathProvider>().As<IPluginListingsPathProvider>();
             builder.RegisterType<NullCreationClubListingsPathProvider>().As<ICreationClubListingsPathProvider>();
         }
+
+        if (command.BlacklistedMods is not null)
```

Comment:
Does this need an `else` where it registers `InjectedBlacklistedModsProvider` with no mods?  I think `FilterBlacklistedReports` will fail to resolve otherwise.

Alternatively, we could just not register `FilterBlacklistedReports` if there's nothing blacklisted.  But i'd have to look closer at the wiring to see if that'd be annoying to do

Reply:
It actually shouldn't. It's a implicit so maybe I should add a comment, but the logic in `RunConfigModule` which runs before this will always register something for `IBlacklistedModsProvider`, so we're always safe there.
And if you had an entry in the config file, you don't want to overwrite that in the command logic where you might not find a setting for this where you would then set your blacklisted mods to an empty list again.